minitwit_nodejs/repositories/userRepository.js

```diff
@@ -19,14 +19,24 @@ function getUserID(username) {
     return helper.getSingle(`select user.user_id from user where user.username = ?`, [username]);
 };
 
+/**
+ * Get amount messages from the database.
+ * @param {string} username
+ * @param {string} password
+ * @param {string} email
+ */
+function addUser(username, password, email) {
+    return helper.insert(`insert into user (username, pw_hash, email) values (?, ?, ?)`, [username, password, email]);
+};
+
 /**
  * Get amount messages from the database.
  * @param {int} whoID
  * @param {int} whomID
  */
 function follow(whoID, whomID) {
```

Comment:
I find this who/whom to be a bit misleading. What do you think?
```suggestion
function follow(followerD, followedID) {
```

Reply:
Sure, I agree. Just copied the minitwit, will change